release-notes/1.0/1.0.19189.1-Preview.md

```diff
@@ -0,0 +1,159 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 1.0.19189.1 Preview released 8 July 2019
+
+This update does not add any major features over the previous preview release but includes the below fixes and changes.
+
+### Target Platform Support
+
+* .NET Framework 4.6+
+* .NET Core 2.1+ (Windows x86, Windows x64, Linux, macOS)
+* .NET Standard 2.0+ (Windows x86, Windows x64, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+* System.Data.Common 4.3.0
+* Microsoft.Data.SqlClient.SNI
```

Comment:
Do we also need the version for this one?

Reply:
Yes, its the preview version we released. I can add it.